src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java

```diff
@@ -50,4 +50,11 @@ public String resourceNotSavedHandler(ResourceNotSavedException e) {
         log.error(String.format("ResourceNotSavedException: %s" ,  e.getMessage()));
         return e.getMessage();
     }
+
+    @ExceptionHandler(ResourceNotFoundException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
Why did you choose ``Bad Request`` for ``wallet not found``? Please use ``404`` instead of ``400``.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404